content/php/concepts/string-functions/terms/strtoupper/strtoupper.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'strtoupper()'
+Description: 'Converts each character in a string to uppercase.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`strtoupper()`** function converts all the characters in a string to uppercase.
+
+## Syntax
+
+```pseudo
+strtoupper(string_to_convert);
+```
+
+The `string_to_convert` parameter is the string that will be converted to uppercase.
+
+## Codebyte
```

Comment:
```suggestion
## Example
```

Reply:
generally we include one or more standard examples (depending on the entry) and zero or more codebytes (these are not always part of an entry)